### ISSUE TITLE
need sticky nav

### ISSUE THREAD

**Issue by yiwiz-sai** [h-enk/doks#735]
sometimes sticky nav is necessary,  like this https://www.andymcdonald.scot/

**Comment by h-enk** [comment]
Thanks for your input. Will add both as an option.

**Comment by constantcopyandwork** [comment]
Hi, @h-enk,
Is there any update on the timeline for this? 
> Will add both as an option.
I was able to implement a to top button and set my footer not to float up on short pages. But my sticky navbar only works without the flexSearch active: docs section is messed up otherwise (sidebar scrolling, padding adjustments, results box under sidebar, etc...).
Thanks!

**Comment by h-enk** [comment]
No, unfortunately I can't give you a timeline (I'm way too busy, and have other priorities first).

BTW, my idea is to go back to one navigation bar. So, combine global and section navigation. This would make for a better UX I think, and it will make it easier to implement site wide search and global sticky top navbar a/o.

![navigation-structure-mobile](https://user-images.githubusercontent.com/3902872/169221129-6bb40df2-16bd-44a2-918d-3742d3ec652c.png)

To help you make it work for you now, I think you should update these lines: https://github.com/h-enk/doks/blob/master/assets/scss/common/_global.scss#L125-L162

**Comment by constantcopyandwork** [comment]
Thanks for the tip! I'll look into it.
